build.sbt

```diff
@@ -9,27 +9,27 @@ crossScalaVersions := Nil // crossScalaVersions must be set to Nil on the aggreg
 lazy val bench = project
   .dependsOn(
     `weepickle-tests` % "compile;test",
-    `weejson-upickle`,
   )
   .enablePlugins(JmhPlugin)
   .settings(
     noPublish,
     crossScalaVersions := Seq(scala213, scala3),
     libraryDependencies ++= Seq(
-      "com.fasterxml.jackson.core" % "jackson-databind" % "2.13.0",
-      "com.fasterxml.jackson.dataformat" % "jackson-dataformat-smile" % "2.13.0",
-      "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.13.0",
-      "io.circe" %% "circe-generic" % "0.14.1",
-      "io.circe" %% "circe-parser" % "0.14.1",
-      "org.msgpack" % "jackson-dataformat-msgpack" % "0.9.0",
-      "com.lihaoyi" %% "sourcecode" % "0.2.7",
+      "com.lihaoyi" %% "upickle" % "3.1.3", // need the latest Scala 3 support
+      "com.fasterxml.jackson.core" % "jackson-databind" % "2.15.2",
```

Comment:
I actually wonder if this is updating too far, I notice play-json is only on version 2.14.3. Any idea how closely we need to track them @htmldoug ?

Reply:
(I know databind can get picky about versions, but maybe the “binary compatible” check makes sure this doesn’t go too far?)

Reply:
That is a really good point, and ratcheting back to 2.14.3 might be a good idea, but ratcheting all the way back to 2.11.4 (from `play-json` 2.8.2) or 2.9.8 (from `play-json` 2.7.4) is probably not. Thankfully the direct dependencies on Jackson's `-databind`, `-dataformat-smile`, `-dataformat-msgpack`, and `-module-scala` are limited to the unpublished `bench` project. The only direct Jackson dependencies from published projects are on `-core` (by `weejson`, `weexml`, and `weeyaml`), `-dataformat-xml` (by `weexml`), and `-dataformat-yaml` (by `weeyaml`). However, both `-dataformat-xml` and `-dataformat-yaml` pull in `-databind` transitively, but I don't believe `weexml` and `weeyaml` are widely used, and if someone needs to depend on the older version, they can exclude/override the transitive one.